docs/articles/licensing/index.rst

```diff
@@ -24,4 +24,35 @@ compliance tooling will be using in the future, and indeed already is
 used by tool makers such as WindRiver and Black Duck, they are included
 both for support for tooling and future proofing the format.
 
+Excluding incompatible licences
+===============================
+
+Business reasons sometimes impose restrictions on the licensing and require that the built image
+contains no software distributed under a specific license. Yocto provides a simple way to enforce
+this, by means of the ``INCOMPATIBLE_LICENCE`` variable which can be added to your image's recipe:
+
+.. code-block:: none
+
+    # A list of licenses, named according to the SPDX standard
+    INCOMPATIBLE_LICENSE = "GPL-3.0 LGPL-3.0 AGPL-3.0 MPL-1.1"
+
+This will exclude all recipes carrying software with these licenses; if other recipes depend on
+these packages, they will also be excluded from the image. In case the image cannot be built due to
+unsatisfied dependencies of some required feature or package group, you will get an error from
+bitbake.
+
+The problem can be solved either by explicitly removing the unsatisfied feature or package group,
+like this:
+
+.. code-block:: none
+
+    IMAGE_INSTALL_remove = "packagegroup-bistro-utils"
+    IMAGE_FEATURES_remove = "tools-debug"
+
+or by adding a layer which would provide equivalent packages, licensed under a compatible license.
+For replacing the packages using the GPL-3.0, Yocto offers a ``meta-gplv2`` [#metagplv2]_ layer with older versions
+of many packages, still licensed under the GPL-2.0.
```

Comment:
I would like to see some discussion around the risks that rise from using older versions of various software, such as bugs, security issues etc.

Also, is this really a practice we want to promote?

Reply:
I think this is almost customer specific information. It likely will never be done in PELUX and there exists some documentation on this upstream. Lastly, we like the GPL family of licenses and would prefer to keep them, so I don't think this info is ideal for the SWF blueprint. I likely would be useful internally somewhere because we may need to do this again however.